tools/macos/docker-for-mac/README.md

```diff
@@ -0,0 +1,58 @@
+# Setting up OpenWhisk with Docker for Mac
+
+OpenWhisk will natively be running on a Mac host with Docker for Mac.
+
+# Prerequisites
+
+The following are required to build and deploy OpenWhisk from a Mac host:
+
+- [Docker 1.12.0](https://docs.docker.com/docker-for-mac/)
+- [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+- [Scala 2.11](http://scala-lang.org/download/)
+- [Ansible 2.1.2.0](http://docs.ansible.com/ansible/intro_installation.html)
+
+**Tip** Versions of Docker and Ansible are lower than the latest released versions, the versions used in OpenWhisk are pinned to have stability during continuous integration and deployment.
+
+
+[Homebrew](http://brew.sh/) is an easy way to install all of these and prepare your Mac to build and deploy OpenWhik. The following shell command is provided for your convenience to install `brew` with [Cask](https://github.com/caskroom/homebrew-cask) and bootstraps these to complete the setup. Copy the entire section below and paste it into your terminal to run it.
+
+```
+echo '
+# install homebrew
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+# install cask
+brew tap caskroom/cask
+# install java 8
+brew cask install java
+# install scala
+brew install scala
+# install pip
+sudo easy_install pip
```

Comment:
`brew install --upgrade python`  # would be safer / better
* it avoids relying on the Apple system Python that is know to have issues
* it installs its own (relatively current) pip
* that pip no longer requires the use of `sudo`  (The entire brew project is anti sudo for very good reasons)